### PR TITLE
Refactor/03-timer-presentation

### PR DESCRIPTION
**Pull request by natesheridan**
What (if any) features are you implementing?
  - Added Colored buttons for the radios
  - Added colored start button based on activity type
What (if anything) did you refactor?
  - Reformat time to read correctly
  - Parsed time format to MM:SS
  - 

Were there any issues that arose?
  - formatting time breaks when either value (minutes/seconds) is blank
        - timer is still functional - formatting is a little off.

Is there anything needed from your teammates?
  - None
  
Next intended features
  - Logging past activties
  - Local Storage

### CODE REVIEW

File: Activity.js [hunk 1]
```diff
@@ -16,21 +16,44 @@ class Activity{
...
+            // }
+            // if (minutes<10){
+            //     parsedMinutes = `0${minutes}`
+            // }
+            // if (seconds === 0){
+            //     parsedSeconds = `00`
+            // }
+
+            countdownText.innerText = (`${parsedTime}`);
+            console.log(`${minutes}-${seconds}`)
```
Comment (pthan1) [inline Activity.js:42]: Remove

File: index.html [hunk 2]
```diff
@@ -19,22 +19,22 @@ <h1 id="left-section-header">New Activity</h1>
         <form name="activity_form" onSubmit="return validate(event);" />
           <div class="form-align">
             <h2>Select a category:</h2>
-            <div class="radios">
-              <div id="studyRadio" class="activity-selection">
+            <div class= "radios">
```
Comment (pthan1) [inline index.html:22]: Remove added spaces

**Review by annnuuuh — APPROVED** [review]
Looks great! I think, generally speaking, we will need to carve out time before final merge to make sure our spacing, naming conventions, organization and indentations are consistent and align with styling guides we've been given.

File: styles.css [hunk 3]
```diff
@@ -87,6 +91,48 @@ form , .timer-view {
   border: 1px solid rgb(255, 255, 255);
 }
 
+.activity-selection:nth-child(1):hover{
```
Comment (pthan1) [inline styles.css:94]: Great use of nth-child

**Review by pthan1 — COMMENTED** [review]
Thanks!

File: Activity.js (hunk 1 above)
Comment (natesheridan) [reply Activity.js:42]: ✅

**Pull request merged**